asn1/CommonLedgerProtocol.asn

```diff
@@ -14,6 +14,12 @@ IMPORTS
     Address,
     Timestamp
     FROM InterledgerTypes
+
+    InterledgerPacket
+    FROM InterledgerPacket -- we rename this file from GenericPacket.asn
+
+    BalancePacket
```

Comment:
What is going to be in this `BalancePacket`? Is it just going to be the available liquidity? Or that and min, max, disputed, etc?

The reason we took the get balance call out of CLP was that you want different things for different use cases, and CLP is just the intersection of what you want in all of those uses. 

If 90+% of the protocols that extend CLP are going to extend it with a call to get available liquidity, then I think it should go in the core protocol. However, if in most cases it won't be used then it shouldn't go in there.